search/src/Components/upload/upload_stepper.js

```diff
@@ -89,13 +92,15 @@ class UploadStepper extends React.Component {
             steps: getSteps(this.props.upload_type),
             upload_id: 0,
             images: [],
-            ag_context: {crop_type: "oats"}
+            ag_context: {crop_type: "oats"},
```

Comment:
It should start blank, I think. The `{crop_type: "oats"}` was only intended for demonstration.